src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinJvmTaskExecutor.kt

```diff
@@ -106,14 +109,24 @@ class KotlinJvmTaskExecutor @Inject internal constructor(
 
         if (outputs.jar.isNotEmpty()) {
           context.execute("create jar", ::createOutputJar)
-          context.execute("produce src jar", ::produceSourceJar)
+          // TODO: Remove - source jar is generated in a separate action
+//          context.execute("produce src jar", ::produceSourceJar)
```

Comment:
To be removed, unless you want to preserve the in-process stuff for safety?

Reply:
Moving the source jar generation seems safe enough.

Reply:
Sounds good. I'll keep this change outside of the flag then, but keep the original flow and have this new action flow behind afore mentioned flag. Sound good?

Reply:
Actually, what do you think if I break the source jar generation into a separate PR and remove all traces of source jar packing from the Kotlin builder, might be easier to reason about these as separate changes?

Reply:
Nevermind, I updated this PR to pull source jar + jdeps generation to their own actions.